### DISCUSSION TITLE
sBTC Research - Recovery Mode

### DISCUSSION THREAD

**Discussion by AshtonStephens** [stacks-archive/sbtc-developer-release#312] [Ideas]
This is a research discussion post meant to be edited with research notes surrounding recovery mode.

The primary concerns are:
- Behavior when stackers never come online.
- Any redefining that needs to be included so that sBTC is truely fault tolerant on first release.

**Comment by AshtonStephens** [comment]
Currently under investigation by @jferrant

**Comment by jferrant** [comment]
Per discussions with Jude. 

The intention of recovery mode is to revert to pre Nakamato block production rules. No stackers required. PoX payouts will instead of being directed to stackers be directed to each pending withdrawal address until return to normal mode (either payouts are completed by this method or the stackers complete it)

This should not be included in this SIP. And my fears about how we would ever actually implement recovery mode to enable stackers to go offline are alleviated.  No action required.
